repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -107,7 +107,7 @@ class ClassLoaderTest extends ReposeValveTest {
 
         then: "the request should bomb"
         mc.handlings.size() == 0
-        mc.receivedResponse.code == "500"
+        mc.receivedResponse.code == "502"
```

Comment:
This is the status code returned when an exception is thrown by the `doFilter` method of a filter. I just wanted to call attention to this change so that we know it has changed.